specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -1182,6 +1182,36 @@
         }
       }
     },
+    "TeradataTableDataset": {
```

Comment:
Some of new definitions added in this PR are not referenced anywhere in [datafactory.json](https://github.com/Azure/azure-rest-api-specs/blob/4e5492538c6ff7831c35875de8b7e5b3ce75961a/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json). What's the purpose to add them?

Reply:
This is used to help customer create Teradata Table Dataset. It is based on "Dataset", and it will be used in pipeline.